portability-core/src/test/java/org/dataportabilityproject/serviceProviders/flickr/FlickrPhotoServiceTest.java

```diff
@@ -0,0 +1,126 @@
+package org.dataportabilityproject.serviceProviders.flickr;
+
+import static com.google.common.truth.Truth.assertThat;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.ArgumentMatchers.anyInt;
+import static org.mockito.ArgumentMatchers.anyString;
+
+import com.flickr4java.flickr.Flickr;
+import com.flickr4java.flickr.FlickrException;
+import com.flickr4java.flickr.auth.Auth;
+import com.flickr4java.flickr.auth.Permission;
+import com.flickr4java.flickr.people.User;
+import com.flickr4java.flickr.photos.Photo;
+import com.flickr4java.flickr.photos.PhotosInterface;
+import com.flickr4java.flickr.photos.Size;
+import com.flickr4java.flickr.photosets.Photoset;
+import com.flickr4java.flickr.photosets.Photosets;
+import com.flickr4java.flickr.photosets.PhotosetsInterface;
+import com.flickr4java.flickr.uploader.Uploader;
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
+import org.dataportabilityproject.cloud.interfaces.JobDataCache;
+import org.dataportabilityproject.dataModels.ExportInformation;
+import org.dataportabilityproject.dataModels.photos.PhotoAlbum;
+import org.dataportabilityproject.dataModels.photos.PhotoModel;
+import org.dataportabilityproject.dataModels.photos.PhotosModelWrapper;
+import org.junit.Test;
+import org.mockito.Mockito;
+
+public class FlickrPhotoServiceTest {
+
```

Comment:
All these members should be private if not currently used outside the class.

Reply:
Done.